x-pack/filebeat/module/coredns/log/test/coredns.log

```diff
@@ -0,0 +1 @@
+2019-03-06T08:55:28.903Z [INFO] 172.17.0.4:36413 - 21583 "A IN httpbin.org.cluster.local. udp 43 false 512" NXDOMAIN qr,rd,ra 136 0.000102078s
```

Comment:
Would be great to have at least 2-3 example events for the test logs to have a bit of variation for testing. Also for the other test log.

Reply:
Sure, will do.